tests/integration/scripts/test_validate_integration.py

```diff
@@ -68,25 +68,26 @@ async def test_revalidate_gets_old_tvars(
         }
     })
 
-    wf_run_dir = cylc_install(src_dir) / 'runN'
+    wf_id = install(src_dir)
+    installed_dir = run_dir / wf_id / 'runN'
 
     # Check that the original installation validates OK:
-    validate(wf_run_dir, against_source=True)
+    validate(installed_dir)
 
     # # Start a scheduler with tvars option:
     schd = scheduler(
-        str(wf_run_dir.relative_to(run_dir)),
+        wf_id + '/runN',
```

Comment:
I think `install` should return the full workflow ID (which would include the run number) to save tests having to manually bodge it like this.